File: src/TIL_0708/Solution_d3_10059_유효기간.py
T = int(input())
list = []
for tc in range(1, T+1):
    N = input()
    list.append(int(N))
# end of input

month1 = [1, 3, 5, 7, 8, 10, 12]
month2 = [2, 4, 6, 9, 11]
for x in list:
    AB = int(x/100)
    CD = int(x%100)
    if(0<AB<13 & 0<CD<13):
        print("AMBIGUOUS")
    elif(0<AB<13):
        print("MMYY")
    elif(0<CD<13):
        print("YYMM")
#end of output

File: src/TIL_0708/Solution_d3_10059_유효기간.java
package TIL_0708;
import java.util.*;

public class Solution_d3_10059_유효기간 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int T = sc.nextInt();
		for(int t=1; t<=T; t++) {
			int N = sc.nextInt();
			int AB = N/100;
			int CD = N%100;
			String ans = "";
			if(0<AB && AB<13 && 0<CD && CD<13) {
				ans = "AMBIGUOUS";
			}
			else if(0<AB && AB<13) {
				ans = "MMYY";
			}
			else if(0<CD && CD<13) {
				ans = "YYMM";
			}
			else {
				ans = "NA";
			}
			System.out.println("#" + t + " " + ans);
		}//end of tc
		
	}//end of main

}//end of class
